apps/backoffice-v2/src/common/utils/handle-zod-error/handle-zod-error.ts

```diff
@@ -2,7 +2,7 @@ import { isZodError } from '../is-zod-error/is-zod-error';
 
 export const handleZodError = <TData>(error: unknown, data: TData) => {
   if (!error || isZodError(error)) {
-    return data;
+    return data ?? null;
```

Comment:
Fixes long-standing issue.  `react-query` is expecting `null` and warns against `undefined`.